lib/stat_tracker.rb

```diff
@@ -1,97 +1,121 @@
-require 'CSV'
-require './lib/team'
-require './lib/game'
 
-class StatTracker
-
-  def initialize
-    @teams = []
-    @games = []
-  end
 
-# team_id,franchiseId,teamName,abbreviation,Stadium,link
+class StatTracker
+  attr_reader :game_path, :team_path, :game_teams_path
 
-  def load_from_csv(root_path = "./data")
-    CSV.foreach("#{root_path}/teams.csv", headers: true, header_converters: :symbol) do |row|
-      data = {team_id: row[:team_id],
-              franchiseid: row[:franchiseid],
-              teamname: row[:teamname],
-              abbreviation: row[:abbreviation],
-              stadium: row[:stadium],
-              link: row[:link]
-            }
-      @teams << Team.new(data)
-    end
+  def self.from_csv(file_paths)
+    game_path = file_paths[:games]
+    team_path = file_paths[:teams]
+    game_teams_path = file_paths[:game_teams]
 
```

Comment:
removed lots of the methods danny had put in from his method work. BEFORE WE MERGE - make sure to either reinsert dannys methods or make sure that are safe elsewhere!!

Reply:
got em!